pkg/config/ingress.go

```diff
@@ -3,11 +3,17 @@ package config
 const (
 	NamespaceOwnershipStrict  = "Strict"
 	NamespaceOwnershipAllowed = "InterNamespaceAllowed"
+	StatusEnabled             = "Enabled"
```

Comment:
Isn't following an idiomatic way to represent string-derived based enums?
```
const (
    StatusEnabled IngressStatusEnum = "Enabled"
    StatusDisabled                  = "Disabled"
)
```

Reply:
You are right, but the second constant would be untyped. Need to add the type to all of them. Done.